include/ensmallen_bits/adamr/adamr.hpp

```diff
@@ -0,0 +1,170 @@
+/**
+ * @file adamr.hpp
+ * @author Niteya Shah
+ * Declaration of the AdamR optimizer.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_ADAM_ADAMR_HPP
+#define ENSMALLEN_ADAM_ADAMR_HPP
+
+#include <ensmallen_bits/adam/adam_update.hpp>
+#include <ensmallen_bits/sgdr/cyclical_decay.hpp>
+#include <ensmallen_bits/sgd/sgd.hpp>
+#include <ensmallen_bits/adam/adamw_update.hpp>
+
+namespace ens {
+/**
+ * This class is based on the Adam Update where the Optimiser
+ * simulates a new warm-started run/restart once a number of epochs are
+ * performed.
+ *
+ * @code
+ * @article{
+ *   title   = {Decoupled Weight Decay Regularization},
+ *   author  = {{Ilya}, L. and {Frank}, H.},
```

Comment:
`Ilya` and `Frank` are the first names; shouldn't this be `{Loschilov, I. and Hutter, F.}`?